Basic/Binding/Basic/Http/CS/Client/Connected Services/CoreWcf.Samples.Http/ConnectedService.json

```diff
@@ -0,0 +1,17 @@
+{
+  "ExtendedData": {
+    "inputs": [
+      "http://localhost/COREWCFSAMPLES/CALCULATORSERVICE?singleWsdl"
+    ],
```

Comment:
Where did the uppercase url come from?

Reply:
When the service is hosted on IIS/IIS Express, it has an Upper case WSDL metadata URL: http://localhost/COREWCFSAMPLES/CALCULATORSERVICE?wsdl
However, the same service will have a correct metadata URL when hosted on Kestrel server: http://localhost:5000/CalculatorService/basicHttp?wsdl

Not sure if that's expected or due to configuration issue, but if the client use Connected Service to generate client proxy methods from IIS-hosted service, it would point to upper case URL.